ga4gh/protocol.py

```diff
@@ -26,6 +26,17 @@ def convertDatetime(t):
     return int(millis)
 
 
+class ProtocolException(Exception):
+    """
+    Something went wrong in the protocol module
+    """
+
+
+class UnableToDecodeJsonException(Exception):
```

Comment:
I would put it in the ``exceptions`` module, and just not have it inherit from BaseServerException. It's useful to keep all the keep all the exception in one place; we don't have to remember which module what exception comes from then.